ckanext/qgov/common/resources/resource_types.json

```diff
@@ -1,17 +1,24 @@
 {
     "allowed_extensions": [
-        "accdb", "csv", "doc", "docx", "esri", "fgdb", "gdb", "geojson",
-        "geotiff", "gpkg", "gpx", "html", "jp2", "jpeg", "jpg", "json",
-        "kml", "kmz", "mtl", "n3", "obj", "pdf", "png", "ppt", "pptx",
-        "rdf", "rtf", "shp", "sparql", "tab", "tif", "tiff", "ttf",
-        "topojson", "txt", "wfs", "wmts", "xls", "xlsx", "xml", "zip"
+        "accdb", "asc", "cdf", "csv", "doc", "docx", "ecw", "esri",
+        "fgdb", "gdb", "geojson", "geotiff", "gpkg", "gpx", "html",
+        "jp2", "jpeg", "jpg", "json", "kml", "kmz", "mdb", "mtl", "n3",
+        "nc", "obj", "pdf", "png", "ppt", "pptx", "rdf", "rtf", "shp",
+        "sparql", "tab", "tif", "tiff", "topojson", "tsv", "ttf", "txt",
+        "wfs", "wmts", "xls", "xlsx", "xml", "zip"
     ],
     "allowed_overrides": {
         "application/octet-stream": ["*"],
-        "text/plain": ["application/xml", "application/sparql-query"]
+        "text/plain": [
+            "application/sparql-query",
+            "application/x-ascii-grid"
+            "application/xml",
```

Comment:
missing a comma after grid"

Reply:
I know. I didn't assign reviewers because I was waiting until I had a working build first. Cross fingers.

Reply:
best to not create a pr until you have a working build. (then i don't get an email on the repo that a pr has been opened etc)